Ai screens and such

Merpblu opened the pull request:
Ports some cool AI core screens from the Polaris codebase, cheers to the spriters that did these new AI sprites.

## About the Pull Request

Adds in about 17 new AI screens and 5 new crash sprites for new and old AI core screens

## Why It's Good For The Game
The players will have more options for an AI core screen. 
<!-- The players will have more options for an AI core screen. -->

## Did you test it?

<!--.
-->I have done a compile of the code no errors nor warnings to report, but I did not have time to test the game in a playable state.

## Authorship
All of the new AI sprites were from the Polaris codebase and said spriters.
<!--
Please note down any individuals or otherwise inspirations/sources you have for this code if it is ported or adapted.
We recommend using the git blame.
-->

## Changelog

:cl:rscadd
/:cl:

<!--
Common tags:
* rscadd - Adding a feature.
* rscdel - Removing a feature.
* tweak - Changing an existing feature.
* bugfix - Fixing an intended functionality that is not working, or correcting an oversight.
* maptweak - Changing something on a map, or adding a new away site. In 99% of cases, all map changes are maptweak.
* spellcheck - Spelling and grammar fixes.
Uncommon tags:
* admin - Adding, removing or changing administrative tools.
* balance - Changing an existing feature in such a way that it may broadly impact game balance; usually reserved for larger changes.
* soundadd - Adding new sounds, usually covered by rscadd unless you're only adding the sounds themselves.
* sounddel - Ditto as above with rscdel
* imageadd - Adding new icons; same situation as soundadd - usually you're adding something that uses these icons, so this isn't needed
* imagedel - Ditto as above.
* experiment - For experimental changes and tests that are intended to be temporary.
* wip - For works in progress. You probably won't get away with using this one.

Examples were changelog entries are optional/not typically required but encouraged:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation

As a courtesy, for ported PRs, please include if you have the blessing of the author. While not required, we encourage basic decency in porting others' work. It is also sufficient to note that the author has not expressed displeasure at the idea of their work getting ported.
Please refrain from porting works of authors that have expressed displeasure in having their work ported, thank you.
-->

Merpblu committed:
Ai screens and such

Ports some cool AI core screens from the Polaris codebase, cheers to the spriters that did these new ai sprites.

EgorDinamit commented:
I don't think it does anything on its own? You might have to add icon states wherever they are mentioned in the code itself.

Merpblu commented:
> I don't think it does anything on its own? You might have to add icon states wherever they are mentioned in the code itself.

Ah.... I did not know this till now

Merpblu committed:
Merge branch 'vlggms:dev' into dev

Merpblu committed:
Ai core screen touch ups.

Places in the code that I forgot to place in with the new AI core screen icons.

EgorDinamit reviewed (changes requested):
There are some issues with rebasing/merging apparently, you deleted some of the newer PRs here.
If you need help I could try and implement AI screens into the game myself.

Merpblu commented:
If you would like to do it, you can.

On Sat, Jul 8, 2023 at 9:28 AM EgorDinamit ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> There are some issues with rebasing/merging apparently, you deleted some
> of the newer PRs here.
> If you need help I could try and implement AI screens into the game myself.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vlggms/tegustation-bay12/pull/390#pullrequestreview-1520713348>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWD2OC6S27L6YUXQ7BVGPKLXPFVCPANCNFSM6AAAAAAZMR74RA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
